src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/storm/bolt/decisionmaker/DecisionMakerHandler.java

```diff
@@ -71,6 +82,19 @@ private void handleTimer(Tuple input) {
         service.tick();
     }
 
+    private void handleLifeCycleEvent(Tuple input) {
```

Comment:
It is mostly copy of `org.openkilda.wfm.AbstractBolt#handleLifeCycleEvent`. If `AbstactBolt` will define `activate` / `deactivate` methods we can use them into successor bolts.

Reply:
Fixed.